static/app/components/events/eventReplay/index.tsx

```diff
@@ -39,7 +41,7 @@ export default function EventReplay({event, group, projectSlug}: Props) {
     return <ReplayClipSection event={event} replayId={replayId} group={group} />;
   }
 
-  if (canShowUpsell && !hasSentOneReplay) {
+  if (canShowUpsell && !hasSentOneReplay && issueTypeConfig.replays.enabled) {
```

Comment:
why is `<EventReplay>` being included if `replays.enabled == false`?

Reply:
we're currently discussing in a slack thread whether this check should be here or not. the check seems mostly related to whether a certain tab should show up in issue details but this panel shows inline on the main event page

Reply:
tagged you in the thread

Reply:
I think what Ryan means is that we could add this check to `GroupEventDetailsContent` instead and skip rendering this component at all if the config returns false. I think I have a slight preference to that since it means no extra requests get sent off if we know we don't need to render it.

Reply:
ahh gotcha, i'll update the callsite